hackathon_site/dashboard/frontend/src/components/general/PopupModal/PopupModal.tsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import Modal from "@material-ui/core/Modal";
+import Button from "@material-ui/core/Button";
+import styles from "components/general/PopupModal/PopupModal.module.scss";
+import { Box } from "@material-ui/core";
+
+interface PopupModalProps {
+    title?: string;
+    description: string;
+    isVisible: boolean;
+    submitText?: string;
+    cancelText?: string;
+    submitHandler(): any;
+    cancelHandler?(): any;
```

Comment:
I messed up the ticket, submitHandler should be optional, and can you also display the submit button only if submitHandler function is not null

Reply:
done!